templates/signup.html

```diff
@@ -3,25 +3,59 @@
 
 <head>
     <title>Sign Up</title>
+    <link rel="stylesheet" href={{ url_for( 'static' , filename='style.css' ) }} />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
```

Comment:
I'm not sure if this is a "problem", but I'm pretty sure I read somewhere that this line should go at the end of your body.

Reply:
Oh yeah you are right. I'll fix this thank you!